force-app/main/default/objects/Session_Registration__c/validationRules/Capacity_Check.validationRule-meta.xml

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Capacity_Check</fullName>
+    <active>true</active>
+    <errorConditionFormula>Session__r.Room_Capacity__c  =  Session__r.Number_of_Registered_Attendee__c</errorConditionFormula>
+    <errorMessage>The capacity is full for this session. There is no additional seat for a new attendee</errorMessage>
+</ValidationRule>
```

Comment:
@bmtemakina Does "Room Capacity" field include number of room capacity for VIPs?

Reply:
No it doesn't

Reply:
I think it will when VIP registration is implemented.

Reply:
So if we say "Room Capacity" will include the number of room capacity for VIPs and then do the validation for only regular attendees just for now until VIP registration implementation. After the implementation update the validation for to be including the VIP check too. Will it be okay for you?
(Validation should be like this until VIP check implementation : Session__r.Room_Capacity__c - Session__r.Room_Capacity_For_VIPs__c = Session__r.Number_of_Registered_Attendee__c)